ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ImmutableDB/Stream.hs

```diff
@@ -3,22 +3,29 @@
 {-# LANGUAGE RecordWildCards     #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 
-module Ouroboros.Consensus.Storage.LedgerDB.Stream (
-    NextBlock (..)
+module Ouroboros.Consensus.Storage.ImmutableDB.Stream (
+    NextItem (..)
   , StreamAPI (..)
+  , streamAPI
+  , streamAPI'
   , streamAll
   ) where
 
 import           Control.Monad.Except
 import           GHC.Stack
 import           Ouroboros.Consensus.Block
+import           Ouroboros.Consensus.Storage.Common
+import           Ouroboros.Consensus.Storage.ImmutableDB hiding (streamAll)
+import qualified Ouroboros.Consensus.Storage.ImmutableDB.API as ImmutableDB
+import           Ouroboros.Consensus.Util.IOLike
+import           Ouroboros.Consensus.Util.ResourceRegistry
 
 {-------------------------------------------------------------------------------
   Abstraction over the streaming API provided by the Chain DB
 -------------------------------------------------------------------------------}
 
 -- | Next block returned during streaming
-data NextBlock blk = NoMoreBlocks | NextBlock blk
+data NextItem blk = NoMoreItems | NextItem blk
 
 -- | Stream blocks from the immutable DB
```

Comment:
```suggestion
-- | Stream items from the immutable DB
```

and a few other occurrences in the Haddock?

Reply:
Done